src/components/Stepper/AddEventForm/AddEventForm.tsx

```diff
@@ -1,5 +1,172 @@
+        ),
+        foodAndBeverages: t(
+          "pages.addEventForm.addEventModal.foodAndBeverages",
+        ),
+        foodBeingSoldLabel: t(
+          "pages.addEventForm.addEventModal.foodBeingSoldLabel",
+        ),
+        foodByThirdPartyLabel: t(
+          "pages.addEventForm.addEventModal.foodByThirdPartyLabel",
+        ),
+        alcoholCoverageLabel: t(
+          "pages.addEventForm.addEventModal.alcoholCoverageLabel",
+        ),
+        transport: t("pages.addEventForm.addEventModal.transport"),
+        driverLicenceLabel: t(
+          "pages.addEventForm.addEventModal.driverLicenceLabel",
+        ),
+        selfTransportation: t(
+          "pages.addEventForm.addEventModal.selfTransportation",
+        ),
+        rentalVehicleOwnage: t(
+          "pages.addEventForm.addEventModal.rentalVehicleOwnage",
+        ),
+        yes: t("pages.addEventForm.addEventModal.yes"),
+        no: t("pages.addEventForm.addEventModal.no"),
+        confirm: t("pages.addEventForm.addEventModal.confirm"),
+        daily: t("pages.addEventForm.addEventModal.daily"),
+        weekly: t("pages.addEventForm.addEventModal.weekly"),
+        monthly: t("pages.addEventForm.addEventModal.monthly"),
+        addEventIconAltText: t("common.iconAltText.addEvent"),
+        errorKeys: {
+          fieldRequired:
+            "pages.addEventForm.addEventModal.errors.fieldRequired",
+          endDateInvalid:
+            "pages.addEventForm.addEventModal.errors.endDateInvalid",
+          endTimeInvalid:
+            "pages.addEventForm.addEventModal.errors.endTimeInvalid",
+          startDateInvalid:
+            "pages.addEventForm.addEventModal.errors.startDateInvalid",
+          startTimeInvalid:
+            "pages.addEventForm.addEventModal.errors.startTimeInvalid",
+        } as { [key: string]: string },
+      },
+    },
+  };
+
+  return (
+    <form className="flex flex-col gap-y-6 lg:mb-64 lg:mr-auto lg:w-4/5">
+      <div className="space-y-2">
+        <h2 className="px-4 pt-6 font-segoe text-3xl font-bold text-primary lg:pt-0">
+          {pageContent.pageTitle}
+        </h2>
+        <p className="px-4 text-lg font-normal text-graphite-700">
+          {pageContent.pageSubTitle}
+        </p>
+      </div>
+      <div>
+        <EventPolicy
+          onAddEventClick={onAddEventClick}
+          translationContent={pageContent.eventPolicies}
+        />
+      </div>
+
+      <AddEventModal
+        eventName={selectedEventName}
+        translationContent={pageContent.addEventForm.addEventModal}
+        onConfirm={() => null}
+        isOpen={showAddEventModal}
+        toggle={toggleAddEventModal}
+      />
+      <div className="flex gap-x-4 px-4 pb-10 pt-2 lg:px-0 lg:pb-0 lg:pt-0">
+        <Button
+          className="lg:w flex items-center justify-center gap-0 rounded-md border-2 border-primary bg-transparent !text-primary hover:bg-primary hover:text-white"
```

Comment:
Button component has default styling based on variants. I'm wondering if we could leverage that instead of writing so many classes for this Button.